junit-platform-launcher/src/main/java/org/junit/platform/launcher/listeners/TestExecutionSummary.java

```diff
@@ -127,4 +129,9 @@
 	 */
 	void printFailuresTo(PrintWriter writer);
 
+	/**
+	 * Get a list of the failures of this test execution.
+	 */
+	List<Failure> getFailures();
+
```

Comment:
Please create a nested interface for `Failure` inside this interface (see https://github.com/junit-team/junit5/pull/552#issuecomment-257602703).